src/custom/components/SearchModal/ManageLists/ManageListsMod.tsx

```diff
@@ -33,6 +33,7 @@ import { ListRowProps } from '.'
 import { useActiveWeb3React } from 'hooks/web3'
 import { DEFAULT_NETWORK_FOR_LISTS } from 'constants/lists'
 import { supportedChainId } from 'utils/supportedChainId'
+import { RowWrapper } from './index' // mod
```

Comment:
Nitpick: Does this work?

```suggestion
import { RowWrapper } from '.' // mod
```

Reply:
Yes it does :) Updated.